### PR TITLE
docs: Add doc for L3 transport.

### PR DESCRIPTION
**Pull request by pshelar**
Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by themarwhal** [comment]
hey @pshelar ! looks like the doc precommit check is failing. See here on how to resolve: https://magma.github.io/magma/docs/next/docs/docs_overview#precommit

**Comment by pshelar** [comment]
> hey @pshelar ! looks like the doc precommit check is failing. See here on how to resolve: https://magma.github.io/magma/docs/next/docs/docs_overview#precommit

fixed it.
Thanks.

File: docs/readmes/howtos/l3_transport.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+id: l3_transport
+title: L3 transport for AGW
+hide_title: true
+---
+# Configuration of SGi L3 transport
+
+Magma AGW has configuration option to create and provision wireguard tunnel
+for PGW traffic, This feature is targeted for inbound roaming deployment.
+
+## **Introduction**
```
Comment (hcgatewood) [inline docs/readmes/howtos/l3_transport.md]: Nit: no need to `**` headers

**Review by hcgatewood — APPROVED** [review]
Lgtm, thanks for adding!

**Pull request merged**